crates/interpreter/src/exec.rs

```diff
@@ -1617,3 +1716,11 @@ fn memory_too_small(x: usize, n: usize, mem: &Memory) {
     #[cfg(feature = "debug")]
     eprintln!("Memory too small: {x} + {n} > {}", mem.len());
 }
+
+fn index_in_side_tables(module: &Module, index: u32) -> Option<usize> {
+    if module.side_table(index as usize).is_empty() {
```

Comment:
Why does it matter whether the side-table is empty? It's fine for a function to not have any jumps (and thus an empty table).

Reply:
It was necessary when the side table computation was done in `Module`. Will try removing it after the lifetime issue is fixed.